src/infrastructure/github.ts

```diff
@@ -74,30 +75,32 @@ export class GitHubClient {
   // commit. Hardcode the (stable) name and email so that we can also author commits
   // as the GitHub actions bot.
   // See https://github.com/orgs/community/discussions/26560#discussioncomment-3252340.
-  public static readonly GITHUB_ACTIONS_BOT: User = {
+  public static readonly GITHUB_ACTIONS_BOT: Readonly<User> = {
     id: 41898282,
-    username: "github-actions[bot]",
+    login: "github-actions[bot]",
```

Comment:
Just curious. Why the change from `username` to `login`?

Reply:
In this case the two layers have different ideas of what a `User` is. In the infrastructure layer it adopts GitHub's model. In the domain layer it uses a simpler model with slightly different terminology.